README.md

```diff
@@ -18,3 +18,12 @@ pip install nox
 nox --list  # list available sessions
 nox --python 3.10 -s tests # run tests session for a specific python version
 ```
+
+### Testing against a real caikit+tgis stack
+
+Tests are run against a mocked instance of a TGIS backend by default. To test against a real
+caikit+tgis stack, it is sufficient to run using the `--real-caikit` flag when running `pytest`:
+
+```bash
+nox -s tests -- --real-caikit tests
```

Comment:
What do you think about adding a blurb about predownloadding the required images and having a minimum docker version installed?

Reply:
Done